addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -360,6 +360,17 @@ def message_new(self, msg, custom_values=None):
             through message_process.
             This override updates the document according to the email.
         """
+
+        # Check if already exists an application for the email address
+        # and post a message to every application found matching the email
+        existing = self.search([
+            '|',
+            ('email_from', 'ilike', '<' + msg.get('from') + '>'),
+            ('email_from', '=', msg.get('from'))])
+        if existing:
+            return existing.message_post(subtype='mail.mt_note', **msg).ids
```

Comment:
In master, the method `message_new` is supposed to return a recordset, not ids :wink: